packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.tsx

```diff
@@ -43,7 +43,7 @@ export type MarkdownSynapseProps = {
   renderInline?: boolean
   objectType?: ObjectType
   loadingSkeletonRowCount?: number
-  onMarkdownProcessingDone?: (ref: HTMLInputElement | null) => void
+  onMarkdownProcessingDone?: (textContent: string | null | undefined) => void
```

Comment:
Instead of passing around the ref, just pass back the plain text version of the processed markdown